compiler/modules/front.syntax/scan.bal

```diff
@@ -133,7 +133,8 @@ final readonly & Keyword[] keywords = [
     "type",
     "typedesc",
     "while",
-    "xml"
+    "xml",
+    "table"
```

Comment:
Do you notice anything about the other values in this list?

Reply:
should they be put in alphabetical order? I'll change it, thanks